app/schemas/ephemera_schema.rb

```diff
@@ -1,22 +1,21 @@
 class EphemeraSchema < ActiveTriples::Schema
   # Administrative
-  property :barcode, predicate: ::RDF::URI("http://plum.com/predicates/barCode")
-  property :folder_number, predicate: ::RDF::URI("http://plum.com/predicates/folder_number")
-  property :genre, predicate: ::RDF::URI("http://plum.com/predicates/genre")
-  property :width, predicate: ::RDF::URI("http://plum.com/predicates/width_cm")
-  property :height, predicate: ::RDF::URI("http://plum.com/predicates/height_cm")
-  property :page_count, predicate: ::RDF::URI("http://plum.com/predicates/pages")
+  property :barcode, predicate: ::RDF::Vocab::DC.identifier
+  property :folder_number, predicate: ::RDF::RDFS.label
+  property :width, predicate: ::RDF::Vocab::SCHEMA.width
+  property :height, predicate: ::RDF::Vocab::SCHEMA.height
+  property :page_count, predicate: ::RDF::Vocab::NFO.pageCount
 
   # Descriptive
   # Language/Title from BasicMetadata
-  property :sort_title, predicate: ::RDF::URI("http://plum.com/predicates/sort_title")
-  property :alternative_title, predicate: ::RDF::URI("http://plum.com/predicates/alternative_title")
-  property :series, predicate: ::RDF::URI("http://plum.com/predicates/series")
+  property :sort_title, predicate: ::OpaqueMods.titleForSort
+  property :alternative_title, predicate: ::RDF::Vocab::DC.alternative
+  property :series, predicate: ::PULTerms.seriesTitle
   # Creator/Contributor/Publisher from BasicMetadata
-  property :geographic_origin, predicate: ::RDF::URI("http://plum.com/predicates/geo_origin")
+  property :geographic_origin, predicate: ::RDF::Vocab::BF2.originPlace
```

Comment:
Should be fine. At the bare minimum we can make them URIs if we broadcast the RDF.